odoo/tests/common.py

```diff
@@ -2427,28 +2427,60 @@ def tagged(*tags):
     def tags_decorator(obj):
         include = {t for t in tags if not t.startswith('-')}
         exclude = {t[1:] for t in tags if t.startswith('-')}
-        obj.test_tags = (getattr(obj, 'test_tags', set()) | include) - exclude # todo remove getattr in master since we want to limmit tagged to BaseCase and always have +standard tag
+        obj.test_tags = (getattr(obj, 'test_tags', OrderedSet()) | include) - exclude # todo remove getattr in master since we want to limmit tagged to BaseCase and always have +standard tag
```

Comment:
`OrderedSet` is not really useful here (`set` is good enough).
In case you keep the `OrderedSet`, then `include` should be an `OrderedSet` as well.